### PR TITLE
mpv, pw-cat: update page; mpv: add pt_BR translation

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/mpv.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Um tocador de vídeo/audio baseado no MPlayer.
+> Mais informações: <https://mpv.io>.
+
+- Tocar um vídeo ou áudio de uma URL ou arquivo:
+
+`mpv {{https://www.youtube.com/watch?v=dQw4w9WgXcQ|caminho/para/arquivo}}'`
+
+- Avançar/retroceder 5 segundos:
+
+`ESQUERDA <ou> DIREITA`
```
Comment (kbdharun) [inline pages.pt_BR/common/mpv.md:12]: @sebastiaanspeck What do you suggest doing in this case?

The command contains a keyboard shortcut which depending on the keyboard layout will differ in other languages, should we go with the English version to fix the errors? (Anything works for me)

File: pages/common/mpv.md [hunk 2]
```diff
@@ -3,13 +3,9 @@
...
 
-- Play a video or audio file:
+- Play a video or audio from a URL or file:
 
-`mpv {{path/to/file}}`
-
-- Play a video or audio file from a URL:
-
-`mpv '{{https://www.youtube.com/watch?v=dQw4w9WgXcQ}}'`
+`mpv {{https://www.youtube.com/watch?v=dQw4w9WgXcQ|path/to/file}}'`
```
Comment (kbdharun) [inline pages/common/mpv.md:8]:
```suggestion
`mpv {{url|path/to/file}}'`
```

File: pages.pt_BR/common/mpv.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# mpv
+
+> Um tocador de vídeo/audio baseado no MPlayer.
+> Mais informações: <https://mpv.io>.
+
+- Tocar um vídeo ou áudio de uma URL ou arquivo:
+
+`mpv {{https://www.youtube.com/watch?v=dQw4w9WgXcQ|caminho/para/arquivo}}'`
```
Comment (kbdharun) [inline pages.pt_BR/common/mpv.md:8]:
```suggestion
`mpv {{url|caminho/para/arquivo}}'`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> For the URL, I would suggest going with a generic placeholder to highlight the file option better, even though having a Rickroll is cool 😄 (it makes it long) .

**Review by kbdharun — APPROVED** [review]

File: pages.pt_BR/common/mpv.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# mpv
+
+> Um tocador de vídeo/audio baseado no MPlayer.
+> Mais informações: <https://mpv.io>.
+
+- Tocar um vídeo ou áudio de uma URL ou arquivo:
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:6]:
```suggestion
- Toca um vídeo ou áudio de uma URL ou arquivo:
```

File: pages.pt_BR/common/mpv.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`mpv {{url|caminho/para/arquivo}}'`
+
+- Avançar/retroceder 5 segundos:
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:10]:
```suggestion
- Avança/retrocede 5 segundos:
```

File: pages.pt_BR/common/mpv.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`LEFT <or> RIGHT`
+
+- Avançar/retroceder 1 minuto:
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:14]:
```suggestion
- Avança/retrocede 1 minuto:
```

File: pages.pt_BR/common/mpv.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`DOWN <or> UP`
+
+- Aumentar ou diminuir a velocidade de reprodução em 10%:
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:18]:
```suggestion
- Aumenta ou diminui a velocidade de reprodução em 10%:
```

File: pages/common/mpv.md [hunk 8]
```diff
@@ -23,9 +19,13 @@
 
 `[ <or> ]`
 
-- Play a file at a specified speed (0.01 to 100, default 1):
+- Take a screenshot of the current frame (saved to `mpv-shotNNNN.jpg` by default):
```
Comment (isaacvicente) [inline pages/common/mpv.md:22]:
```suggestion
- Take a screenshot of the current frame (saved to `./mpv-shotNNNN.jpg` by default):
```

File: pages.pt_BR/common/mpv.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`[ <or> ]`
+
+- Capturar a imagem atual (salva em `mpv-shotNNNN.jpg` por padrão):
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:22]:
```suggestion
- Captura a imagem atual (salva em `./mpv-shotNNNN.jpg` por padrão):
```

File: pages.pt_BR/common/mpv.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+`s`
+
+- Tocar um arquivo em uma velocidade especificada (1 por padrão):
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:26]:
```suggestion
- Toca um arquivo em uma velocidade especificada (1 por padrão):
```

File: pages.pt_BR/common/mpv.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+`mpv --speed {{0.01..100}} {{caminho/para/arquivo}}`
+
+- Tocar um arquivo usando um perfil definido no arquivo `mpv.conf`:
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:30]:
```suggestion
- Toca um arquivo usando um perfil definido no arquivo `mpv.conf`:
```

File: pages.pt_BR/common/mpv.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+`mpv --profile {{nome_do_perfil}} {{caminho/para/arquivo}}`
+
+- Mostrar a saída da webcam ou de outro dispositivo de entrada de vídeo:
```
Comment (isaacvicente) [inline pages.pt_BR/common/mpv.md:34]:
```suggestion
- Mostra a saída da webcam ou de outro dispositivo de entrada de vídeo:
```

**Review by isaacvicente — CHANGES_REQUESTED** [review]
Olá! Muito obrigado pela sua contribuição! :tada: 

Sugeri algumas mudanças sobre o verbo usado nas descrições. O verbo tem que estar no presente do indicativo, ou seja, o verbo _explica_ ou _descreve_ o que o comando faz.

Espero revisar mais traduções suas! Valeu!

**Review by isaacvicente — APPROVED** [review]
Tradução pronta pro merge! Obrigado mais uma vez por traduzir :100:

**Pull request merged**